ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.16.yaml

```diff
@@ -85,6 +85,20 @@ tests:
   steps:
     cluster_profile: ibmcloud-qe-2
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-disconnected-private
+- as: installer-rehearse-vsphere
```

Comment:
To regularly run this job, I think this job should be added into ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.N.yaml

Reply:
this is not a new feature, only a polarion case. why we also need added it in daily job?

Reply:
ok. thanks for clarification.